config/config.json

```diff
@@ -220,4 +220,4 @@
         "InterNodeListenAddress": ":8075",
         "InterNodeUrls": []
     }
-}
+}
```

Comment:
minor thing, not blocking anything but it would be nice if nothing has changed that the file remains the same